app/src/main/java/com/mapbox/vision/examples/activity/ar/ArMapActivity.kt

```diff
@@ -160,23 +176,25 @@ class ArMapActivity : AppCompatActivity(), MapboxMap.OnMapClickListener,
             })
     }
 
-    private fun enableLocationPlugin() {
+    private fun enableLocationComponent() {
         initializeLocationEngine()
-        locationLayerPlugin = LocationLayerPlugin(mapView, mapboxMap).apply {
-            cameraMode = CameraMode.TRACKING
-            lifecycle.addObserver(this)
-        }
+
+        val locationComponentActivationOptions = LocationComponentActivationOptions
+                .builder(this, mapboxMap.style!!)
```

Comment:
is `mapboxMap.style!!` Int? may be set some default value instead unsafe `!!`?

Reply:
`mapboxMap.style` is `Style?`, but it is not null because it is set on a previous step